src/NuGet.Core/NuGet.Protocol.Core.Types/NuGetHttpHeaderUtility.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Net.Http;
+using NuGet.Common;
+
+namespace NuGet.Protocol.Core.Types
+{
+    public class NuGetHttpHeaderUtility
+    {
+        public const string ApiKeyHeader = "X-NuGet-ApiKey";
+        public const string WarningHeader = "NuGet-Warning";
```

Comment:
nit: in the client we normally do *public static readonly* instead of public const incase it ever changes (which this shouldn't). internal or private const is fine.